11-ras-vec.Rmd

```diff
@@ -131,39 +143,50 @@ rast_poly_point = tm_shape(srtm) +
 rast_poly_point
 ```
 
-The second example shows raster extraction along a line or lines.
-For this purpose, we will create a simple line going from northwest to southeast of the Zion National Park (\@ref(fig:lineextr):A):
+Raster extraction also works with line selectors.
+To demonstrate this, the code below creates `zion_transect`, a straight line going from northwest to southeast of the Zion National Park, illustrated in Figure \@ref(fig:lineextr):A (see section \@ref(vector-data) for a recap on the vector data model):
 
 ```{r}
-zion_transect = st_sfc(st_linestring(rbind(c(-113.2, 37.45), c(-112.9, 37.2)))) %>% 
-  st_sf()
+zion_transect = rbind(c(-113.2, 37.45), c(-112.9, 37.2)) %>%
+  st_linestring(.) %>% 
+  st_sfc(.) %>% 
+  st_sf(.)
+```
+
+
+```{r, eval=FALSE, echo=FALSE}
+# Aim: show how extraction works with non-straight lines by
```

Comment:
@Nowosad are you sure the code also works for non-straight lines? The plot below, generated when this 2 segment line is used, doesn't look right to me - was expecting the elevation profile to be roughly mirrored after the half-way point:

![image](https://user-images.githubusercontent.com/1825120/39084587-31fcea2e-4570-11e8-9f73-9e0183153362.png)